### DISCUSSION TITLE
How often will manim-gl package be updated w.r.t. to the repository?

### DISCUSSION THREAD

**Discussion by baljeetrathi** [3b1b/manim#1465] [General]
Is there a specific criteria for bumping up manim versions on pip?

Maybe we could release a new minor version every month with commits made during the month.

**Comment by TonyCrane** [comment]
But this repo doesn't update very often, so I just plan to release new versions after important bug-fixes.

**Reply by baljeetrathi** [reply]
Thanks @TonyCrane :)

**Comment by mitkonikov** [comment]
@TonyCrane There are major bugs fixed from the latest Manim release (2 years ago). Any plans for releasing a new minor version? If you need someone to test things, message me.
